sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/SwitchDefaultClauseEmpty.cs

```diff
@@ -32,12 +32,51 @@ public SwitchDefaultClauseEmpty(Fruit fruit)
                 default:
                     break;
             }
+
+            switch (fruit)
+            {
+                case Fruit.Apple:
+                    Console.WriteLine("apple");
+                    break;
+                default:
+                    // Single line comment before
+                    break;
+            }
+
+            switch (fruit)
+            {
+                case Fruit.Apple:
+                    Console.WriteLine("apple");
+                    break;
+                default:
+                    break; // Single line comment after
+            }
+
             switch (fruit)
             {
                 case Fruit.Apple:
                     Console.WriteLine("apple");
                     break;
-                case Fruit.Orange: // Noncompliant
+                default:
+                    /* Multi lines comment before */
+                    break;
+            }
+
+            switch (fruit)
+            {
+                case Fruit.Apple:
+                    Console.WriteLine("apple");
+                    break;
+                default:
+                    break; /* Multi lines comment after */
+            }
+
+            switch (fruit)
+            {
+                case Fruit.Apple:
+                    Console.WriteLine("apple");
+                    break;
+                case Fruit.Orange:

```

Comment:
What is the expected behaviour if the comment is on the same line as the _default_ keyword?

Reply:
Good question... Not sure whether this will count as leading trivia of `break` or trailing trivia of `default`. I will add one test.

Reply:
Will it work with:
```
default:
{
  // comment
}
break;
```

Reply:
it works.